static/src/javascripts/bootstraps/enhanced/profile.js

```diff
@@ -37,6 +41,16 @@ const initFormstack = (): void => {
     });
 };
 
+const initWizards = (): void => {
+    const wizards = [
+        ...document.querySelectorAll(`.${wizardContainerClassname}`),
```

Comment:
This could be `getElementsByClassName()` - it's a lot faster. Also: it would be great if you could wrap it into `fastdom` ✨